### PR TITLE
321 packet logs

### PR DESCRIPTION
**Pull request by faeron**
### :dart: Relevant issues
#321 

### :gem: Type of change
- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

## :vertical_traffic_light: How Has This Been Tested?

- I started moleculer with npm dev script and defined transit.packetLogFilter for various packet types (heartbeat, discover, info) to verify that log messages do not appear.
- Unit tests

## :checkered_flag: Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] **I have added tests that prove my fix is effective or that my feature works**
- [x] **New and existing unit tests pass locally with my changes**
- [x] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/transit.js [hunk 1]
```diff
@@ -36,7 +41,7 @@ class Transit {
 		this.logger = broker.getLogger("transit");
 		this.nodeID = broker.nodeID;
 		this.tx = transporter;
-		this.opts = opts;
+		this.opts = _.defaultsDeep(opts, defaultOptions);
```
Comment (icebob) [inline src/transit.js]: Please don't duplicate default values. Use `ServiceBroker.defaultOptions.transit` instead.

**Comment by icebob** [comment]
@faeron Could you fix it?

**Comment by faeron** [comment]
Yup, I will fix it asap.

**Comment by faeron** [comment]
When trying to use ``ServiceBroker.defaultOptions.transit``some more tests are failing. It seems that there could be an issue with circular dependencies (``Transit`` now requires ``ServiceBroker``, which itself requires ``Transit``). Any idea on how to resolve that?

**Comment by icebob** [comment]
Ohh, I see. In this case, drop the defaultOptions in the transit because it gets default options from broker where `packetLogFilter` is set to `[]` by default. 
In brief, just use this option, no need to merge it with default values in contructor.

Thanks!

**Comment by faeron** [comment]
Should ready to get merged.

**Pull request merged**